tests/test.hs

```diff
@@ -48,11 +48,20 @@ import Paths_liquidhaskell
 
 import Text.Printf
 
+-- | Tests which have dependencies can use this record type to express that the
+-- `dependencies` are run *sequentially* in a certain order, and the `toplevel`
+-- tests are run in *parallel* after all dependencies have been run. For more
+-- information on specifying the order of the dependencies, see the
+-- `SequentialFileOrder` sections below; by default they are run in *reverse*
+-- alphabetical order.
 data DependentTests = DependentTests
   { dependencies :: [TestTree]
   , toplevel :: [TestTree]
   }
 
+-- | Tests which are purely sequential (like most benchmarks) can use this
```

Comment:
Is there any reason to run some benchmarks in parallel?
I'm thinking it could help to make all benchmarks sequential to simplify explanations. Also, the measurements would be more comparable across runs if they are built sequentially, though I still would expect VMs to introduce a lot of variability.

Reply:
This is a good point, I will change it.